src/view/treeNodes/pullRequestNode.ts

```diff
@@ -402,13 +403,13 @@ export class PRNode extends TreeNode {
 					return right.join('\n');
 				}
 			} else {
-				if (params.base) {
-					const originalFileName = fileChange.status === GitChangeType.RENAME ? fileChange.previousFileName : fileChange.fileName;
-					return getFileContent(this.repository.path, params.commit, originalFileName);
+				const originalFileName = fileChange.status === GitChangeType.RENAME ? fileChange.previousFileName : fileChange.fileName;
+				const originalFilePath = path.join(this.repository.rootUri.fsPath, originalFileName);
+				const originalContent = await this.repository.show(params.commit, originalFilePath);
```

Comment:
It seems that sometimes the result from the api ends with an additional line break. For example in https://github.com/auchenberg/pullrequest-demo/pull/6/files , `app.js`'s last line is not empty.

Reply:
I can't repro that one. When I open the diff in the PR extension, I get the same diff as in GitHub:

![image](https://user-images.githubusercontent.com/22350/45536068-9e4a9980-b800-11e8-9218-9fcf720cdc8b.png)

Reply:
You can probably see this once you have line numbers turned on. Screenshots from latest binary

<img width="981" alt="image" src="https://user-images.githubusercontent.com/876920/45590199-28b9f880-b8e8-11e8-93d6-5d85eaaf2d4b.png">

GitHub's diff

<img width="1390" alt="image" src="https://user-images.githubusercontent.com/876920/45590204-438c6d00-b8e8-11e8-9c7e-97e8e79cd479.png">

We get a line break at the end so we have line 59 in the diff view.

Reply:
@joaomoreno thanks for the detailed investigation, I verified that the diff view on GitHub doesn't  show the empty line always somehow. @shana do you have insights on this?